### PR TITLE
Add minimum usage for CLI v0.15.x to docs

### PR DESCRIPTION
**Pull request by runephilosof-karnovgroup**
Minimum effort documentation, better than none.
Covers:
* Fetching the tool
* Running the tool
* Basic usage help output

### CODE REVIEW

**Comment by runephilosof-karnovgroup** [comment]
Thanks, those are much more thorough than what I made here.
If [they](https://woodpecker-ci.org/docs/next/cli) are valid for v0.15, they should be shown [there](https://woodpecker-ci.org/docs/cli) as well.
If they are not valid for v0.15, then I think my PR should be merged.
Maybe with an addition of a link to the next version docs.

File: docs/versioned_docs/version-0.15/40-cli.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+```docker run --rm woodpeckerci/woodpecker-cli:v0.15```
+```bash
+NAME:
+   woodpecker-cli - command line utility
+
+USAGE:
+   woodpecker-cli [global options] command [command options] [arguments...]
+
+VERSION:
+   next-fc7aacb6
```
[the remark this answers is not in the document]
Comment (runephilosof-karnovgroup) [reply docs/versioned_docs/version-0.15/40-cli.md:12]: Actually that is the output from the `v0.15` tag.
You see the command I typed and the output I got.
I have the same issue with the `woodpeckerci/woodpecker-server:v0.15` image, it says `next` as version in the upper left.

[the remark this answers is not in the document]
Comment (runephilosof-karnovgroup) [reply docs/versioned_docs/version-0.15/40-cli.md:12]: `fc7aacb6` is the top commit on the `v0.15` branch
https://github.com/woodpecker-ci/woodpecker/commits/release/v0.15

[the remark this answers is not in the document]
Comment (6543) [reply docs/versioned_docs/version-0.15/40-cli.md:12]: oh no ... a bug

[the remark this answers is not in the document]
Comment (6543) [reply docs/versioned_docs/version-0.15/40-cli.md:12]: well we can at least fix this for our docs ;)

Comment (6543) [inline docs/versioned_docs/version-0.15/40-cli.md:12]:
```suggestion
   v0.15.x
```

**Review by 6543 — APPROVED** [review]

File: docs/versioned_docs/version-0.15/40-cli.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+   woodpecker-cli [global options] command [command options] [arguments...]
+
+VERSION:
+   v0.15.x
+
+COMMANDS:
+   build      manage builds
+   log        manage logs
+   deploy     deploy code
+   exec       execute a local build
```
Comment (6543) [inline docs/versioned_docs/version-0.15/40-cli.md:18]:
```suggestion
   exec       execute a pipeline locally
```

File: docs/versioned_docs/version-0.15/40-cli.md [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+   woodpecker-cli - command line utility
+
+USAGE:
+   woodpecker-cli [global options] command [command options] [arguments...]
+
+VERSION:
+   v0.15.x
+
+COMMANDS:
+   build      manage builds
```
Comment (6543) [inline docs/versioned_docs/version-0.15/40-cli.md:15]:
```suggestion
   build      manage pipelines
```

**Pull request merged**